components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenManagementDAOImpl.java

```diff
@@ -198,16 +199,24 @@ public RefreshTokenValidationDataDO validateRefreshToken(String consumerKey, Str
                     validationDataDO.setAccessTokenIssuedTime(
                             resultSet.getTimestamp(13, Calendar.getInstance(TimeZone.getTimeZone(UTC))));
                     validationDataDO.setAccessTokenValidityInMillis(resultSet.getLong(14));
+                    String authorizedOrganization = resultSet.getString(15);
```

Comment:
After the access-token-validity-period, the newly introduced authorized_organization column is added. Due to that below numberings were increased by 1.

<img width="879" alt="Screenshot 2023-10-15 at 15 59 16" src="https://github.com/wso2-extensions/identity-inbound-auth-oauth/assets/35717390/96fd311a-0434-4fcb-a3ae-1e75c0d7dcc4">